doc/install.md

```diff
@@ -206,6 +206,11 @@ Relevant issues: [#134](https://github.com/romanz/electrs/issues/134) and [#391]
 
 #### Dynamic linking
 
+Note that if you have previously done a static linking build, it is recommended to clean the build artifacts to avoid build errors:
```

Comment:
```suggestion
Note that if you have previously done a static linking build, it is recommended to clean the build artifacts to avoid build errors (e.g. https://github.com/romanz/electrs/issues/1001):
```